changelogs/fragments/193.yaml

```diff
@@ -0,0 +1,3 @@
+---
+trivial:
```

Comment:
@Akasurde could you please reference the textfsm bug https://github.com/google/textfsm/issues/105 in the changelog?